src/vs/workbench/services/runtimeStartup/common/runtimeStartup.ts

```diff
@@ -92,6 +92,10 @@ export class RuntimeStartupService extends Disposable implements IRuntimeStartup
 
 	onDidChangeRuntimeStartupPhase: Event<RuntimeStartupPhase>;
 
+	// The event emitter for the onDidCompleteDiscoveryPhase event.
+	onDidCompleteDiscoveryPhaseEmitter =
+		this._register(new Emitter<undefined>);
+
```

Comment:
I think this should be: `new Emitter<void>()`

```
	// The event emitter for the onDidCompleteDiscoveryPhase event.
	onDidCompleteDiscoveryPhaseEmitter =
		this._register(new Emitter<void>());
```

And it can then be fired:

```
	this.onDidCompleteDiscoveryPhaseEmitter.fire();
```